File: modul-4/Soal-3/PRAK403-2310817110013-Randy_Febrian.py
a, b = map(int, input().split())
nilaia, nilaib = a, b

if a > b:
    print("%d %d - " % (a, b), end='')
    for i in range(a - b):
        nilaia -= 1
        nilaib += 1
        if i < a - b - 1:
            print("%d %d - " % (nilaia, nilaib), end='')
        else:
            print("%d %d" % (nilaia, nilaib), end='')
else:
    print("%d %d - " % (a, b), end='')
    for i in range(b - a):
        nilaia += 1
        nilaib -= 1
        if i < b - a - 1:
            print("%d %d - " % (nilaia, nilaib), end='')
        else:
            print("%d %d" % (nilaia, nilaib), end='')

File: modul-4/Soal-3/PRAK403-2310817110013-Randy_Febrian.c
#include <stdio.h>

int main(){
    int a, b;
    scanf("%d %d", &a, &b);
    int nilaia = a, nilaib = b;

    if(a > b){
        printf("%d %d - ", a, b);
        for(int i = 0; i < a - b; i++){
            nilaia--, nilaib++;
            if (i < a - b - 1) {
                printf("%d %d - ", nilaia, nilaib);
            } 
            else {
                printf("%d %d\n", nilaia, nilaib);
            }
        }
    }
    else{
        printf("%d %d - ", a, b);
        for(int i = 0; i < b - a; i++){
            nilaia++, nilaib--;
            if (i < b - a - 1) {
                printf("%d %d - ", nilaia, nilaib);
            } 
            else {
                printf("%d %d\n", nilaia, nilaib);
            }
        }
    }  
    return 0;
}